docs-site/content/guide/high-availability.md

```diff
@@ -272,3 +272,137 @@ final config = Configuration(
 
   </template>
 </Tabs>
+ 
+ ## Deploy a Typesense cluster to Docker Swarm  
+
+To Deploy a Typesense cluster to Docker Swarm, follow these steps:
+
+- Initialize Docker Swarm. For example, in a 4 node setup, target docker engine on one of the nodes to act as a manager
+
+```shell  
+docker swarm init --advertise-addr $(hostname -i)
+Swarm initialized: current node (6082x127zz98f0pwgjexbv5xp) is now a manager.
+   
+To add a worker to this swarm, run the following command:
+
+docker swarm join --token SWMTKN-1-30txqn35hmpwjpk2qq2zmled1rf94pcft2nbhsb0ckleco9pb2-bjh6oh9yz3vk58uimd6v3jjky 192.168.0.25:2377
+
+To add a manager to this swarm, run 'docker swarm join-token manager' and follow the instructions.
+```
+
+- Add docker instance on rest of the nodes to existing Swarm as node. Repeat the below command on all nodes
```

Comment:
Could we create named tabs like this so it's clear that these commands need to be run on different nodes: https://github.com/typesense/typesense-website/blob/a11202d03e553826ddf04918cc57396a849f4872/docs-site/content/guide/high-availability.md#example

Reply:
will do.